assets/stylesheets/bootstrap/_type.scss

```diff
@@ -3,6 +3,19 @@
 // --------------------------------------------------
 
 
+@function em($pixels, $context: $browser-context) {
```

Comment:
I think functions warrant doc blocks here. It's not clear how text scaling is applied. I used to remember this myself but I've forgotten!